app/views/categories/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render partial: 'shared/details_nav' %>
+<div class="container form_styles">
+<h1> Create a new category</h1>
+
+<%= form_with model: @category, url: categories_path do |f| %>
+  <div>
+  <%= f.label :name %><br />
+  <%= f.text_field :name %>
+  </div>
+  <div>
+  <%= f.label :icon %><br />
+  <%= f.text_field :icon %>
+  </div>
+  <%= f.hidden_field :author_id, value: current_user.id if current_user.present? %>
+  <%= f.submit :submit %><br />
+  <% end %>
+</div>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you follow the design guideline provided. Let's ensure that the buttons for the category and transaction page stay at the bottom of the page as it is in the design template provided. This will enhance a great user interface and experience.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="339" alt="Screenshot 2023-04-28 at 19 05 09" src="https://user-images.githubusercontent.com/58692590/235225800-68021fab-941d-4c65-ae9b-76e44b63c9dd.png"> | <img width="441" alt="Screenshot 2023-04-28 at 19 05 34" src="https://user-images.githubusercontent.com/58692590/235225839-f48952cb-853f-4bc8-a783-c521e7895262.png">

Reply:
Corrected

Reply:
-  Well resolved 👍🏽 
<img width="329" alt="Screenshot 2023-04-29 at 14 17 06" src="https://user-images.githubusercontent.com/58692590/235304717-4cff60b7-6188-43ff-9589-e0d9b714656a.png">